packages/form-footer/src/FormFooter.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useMemo } from 'react';
+import Banner from '@leafygreen-ui/banner';
+import Button from '@leafygreen-ui/button';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { isComponentType } from '@leafygreen-ui/lib';
+import { once } from 'lodash';
+
+const footerStyle = (sticky: boolean) => {
+  return css`
+    position: ${sticky ? 'fixed' : 'static'};
+    bottom: ${sticky ? '0px' : 'unset'};
+    left: ${sticky ? '0px' : 'unset'};
+    min-height: 92px;
+    width: 100%;
+    padding: 24px;
+    border: 1px solid ${uiColors.gray.light2};
+    box-shadow: 0px -4px 4px 0px rgba(6, 22, 33, 0.1);
```

Comment:
Is this RGBA value one of our palette colors? Can we use polished to programmatically produce this value instead?